tests/TestCase/Command/MailerCommandTest.php

```diff
@@ -48,8 +48,6 @@ public function testMain()
         $this->generatedFiles = [
             APP . 'Mailer/ExampleMailer.php',
             ROOT . 'tests/TestCase/Mailer/ExampleMailerTest.php',
-            ROOT . 'templates/layout/email/html/example.php',
-            ROOT . 'templates/layout/email/text/example.php',
```

Comment:
Should these maybe be `templates/email/html/example.php` and `templates/email/text/example.php`

Reply:
Creating empty files under templates/email/html and templates/email/text doesn't do much good either.

Reply:
It prevents the user from getting an error when first running it. Right now after running bake you'd kind of expect everything to be configured. Maybe just a docBlock with just 
```
<?php
/**
 * @var \App\View\AppView $this
 */
?>
```

would be helpful

Reply:
> It prevents the user from getting an error when first running it.

What error?

Reply:
```
Cake\View\Exception\MissingTemplateException: Template file 'email/text/contact_us.php' could not be found.

The following paths were searched:

- /Users/challgren/Projects/TruePedigree/website-cake/templates/
- /Users/challgren/Projects/TruePedigree/website-cake/vendor/cakephp/cakephp/templates/

#0 /Users/challgren/Projects/TruePedigree/website-cake/vendor/cakephp/cakephp/src/View/View.php(747): Cake\View\View->_getTemplateFileName('email/text/cont...')
#1 /Users/challgren/Projects/TruePedigree/website-cake/vendor/cakephp/cakephp/src/Mailer/Renderer.php(86): Cake\View\View->render()
#2 /Users/challgren/Projects/TruePedigree/website-cake/vendor/cakephp/cakephp/src/Mailer/Mailer.php(372): Cake\Mailer\Renderer->render('', Array)
#3 /Users/challgren/Projects/TruePedigree/website-cake/vendor/cakephp/cakephp/src/Mailer/Mailer.php(389): Cake\Mailer\Mailer->render('')
#4 /Users/challgren/Projects/TruePedigree/website-cake/vendor/cakephp/cakephp/src/Mailer/Mailer.php(354): Cake\Mailer\Mailer->deliver()
#5 /Users/challgren/Projects/TruePedigree/website-cake/src/Shell/Task/QueueCustomEmailTask.php(60): Cake\Mailer\Mailer->send('contactUs', Array)
#6 /Users/challgren/Projects/TruePedigree/website-cake/vendor/dereuromark/cakephp-queue/src/Shell/QueueShell.php(264): App\Shell\Task\QueueCustomEmailTask->run(Array, 1)
#7 /Users/challgren/Projects/TruePedigree/website-cake/vendor/dereuromark/cakephp-queue/src/Shell/QueueShell.php(214): Queue\Shell\QueueShell->runJob(Object(Queue\Model\Entity\QueuedJob), '21515')
```

Reply:
The generated mailer does not contain any action. So I presume you created a `contactUs()` in your `ContactUsMailer` to get that error, since by default the mailer expects a template matching the method name. 

Personally I have never created a method in the mailer class with the same name as class name.

Reply:
Yep, so I should maybe try using a different mailer name? But maybe a info after baking about creating the templates.

Reply:
Well you can create whatever name you want, just create corresponding template or set template to `default`.